gc/base/standard/ConcurrentGC.cpp

```diff
@@ -3524,36 +3524,34 @@ MM_ConcurrentGC::scanRememberedSet(MM_EnvironmentBase *env)
 					&& _markingScheme->isMarkedOutline(objectPtr)
 					&& !_cardTable->isObjectInDirtyCardNoCheck(env,objectPtr)) {
 						RSObjects += 1;
-						if (_extensions->dirtCardDuringRSScan) {
-							_cardTable->dirtyCard(env, objectPtr);
-						} else {
-							/* VMDESIGN 2048 -- due to barrier elision optimizations, the JIT may not have dirtied
-							 * cards for some objects in the remembered set. Therefore we may discover references
-							 * to both nursery and tenure objects while scanning remembered objects.
-							 */
-
-							bytesTraced += _markingScheme->scanObject(env,objectPtr, SCAN_REASON_REMEMBERED_SET_SCAN);
-
-							/* Have we pushed enough new references? */
-							if(env->_workStack.getPushCount() >= maxPushes) {
-								/* To reduce the chances of mark stack overflow, we do some marking
-								 * of what we have just pushed.
-								 *
-								 * WARNING. If we HALTED concurrent then we will process any remaining
-								 * workpackets at this point. This will make RS processing appear more
-								 * expensive than it really is.
-								 */
-								while(NULL != (objectPtr = (omrobjectptr_t)env->_workStack.popNoWait(env))) {
-									bytesTraced += _markingScheme->scanObject(env, objectPtr, SCAN_REASON_PACKET);
-								}
-								env->_workStack.clearPushCount();
+						/* VMDESIGN 2048 -- due to barrier elision optimizations, the JIT may not have dirtied
```

Comment:
The indentation is off here.

Reply:
Ok, I was just trying to keep the indentation consistent with how it was before. I didn't like it either but left it how it was to be consistent with what was there before, but will fix it now.